code/obj/item/device/transfer_valve.dm

```diff
@@ -529,9 +551,10 @@ TYPEINFO(/obj/item/device/transfer_valve/briefcase)
 			boutput( user, "<b>There's no crystal in this here device!</b>")
 		else
 			if(crystal.pressure)
-				boutput( user, "The reader reads <b>[crystal.pressure/25]</b> kilojoules." )
+				boutput( user, "The reader reads <b>[crystal.pressure]</b> kiloblast." )
```

Comment:
What is a kiloblast?

Reply:
A nonexistent unit (just like joules in _units.dm) :)
Anyway I looked into explosion power unit -> Joule conversion, but it is not worth it. So kiloblast is just 1:1 with an explosion's power value instead